src/python/pants/backend/cc/lint/clangformat/subsystem.py

```diff
@@ -31,19 +26,15 @@ class ClangFormat(PythonToolBase):
         """
     )
 
-    default_version = "clang-format>=14.0.3,<16"
     default_main = ConsoleScript("clang-format")
-    default_requirements = [default_version]
+    default_requirements = ["clang-format>=14.0.3,<16"]
```

Comment:
Note throughout that `default_requirements` is only used by our `generate_builtin_lockfiles.py` script. Nothing else cares about it.